files.txt

```diff
@@ -0,0 +1,18 @@
+BilbyFs_CorresProof.patch
```

Comment:
What do we need this file `files.txt`? By the looks of it you are creating the file in the tests, do you need this checked in?